src/common/util/submit-data.js

```diff
@@ -89,6 +89,13 @@ submitData.img = function img (url) {
 submitData.beacon = function (url, body) {
   // Navigator has to be bound to ensure it does not error in some browsers
   // https://xgwang.me/posts/you-may-not-know-beacon/#it-may-throw-error%2C-be-sure-to-catch
-  const send = window.navigator.sendBeacon.bind(navigator)
-  return send(url, body)
+  const send = window.navigator.sendBeacon.bind(window.navigator)
```

Comment:
Is the change of bind value to `window.navigator` a just-in-case-probably-should? Or does the scope matter somehow to the catch/fallback?